### ISSUE TITLE
Doesn't work anymore

### ISSUE THREAD

**Issue by mainrs** [javdejong/nhk-pronunciation#3]
I setup a fresh install of Anki 2 and the Pitch accent plugin alongside the Japanese Support plugin. My model and my card name has the word 'Japanese' in it. After adding `触る` for example to the Expression field the Pronunciation field is still empty. I verified twice that the names where all correct and I didn't mess anything up.

**Comment by twwn** [comment]
Just to confirm, the plugin won't equip over half of my cards – always the same affected – with the pronunciation, no matter what I try.

On Arch, Anki 2.0.36, python 2.7.12

**Comment by SpongebobSquamirez** [comment]
Can you give an example or two of an expression (please give the exact/entire thing occupying the field) that won't get a pronunciation? The pull request I recently made should fix the vast majority of problems with the plugin, but if there are more problems I might try to tackle those as well.

**Comment by SpongebobSquamirez** [comment]
I've confirmed that my pull request fixes your example (successfully gives the accent for 触る) as well as all the examples from a similar comment [here](https://ankiweb.net/shared/info/932119536) (namely, 基本, 寄付, 義務, 教科書). Once my pull request is merged into this repository I think this issue can be closed.

**Comment by mainrs** [comment]
Looks great so far! Thanks for your work! @weirdalsuperfan

**Comment by javdejong** [comment]
Since #18 is now merged, I'm closing this issue.
